VideoWeb/VideoWeb.AcceptanceTests/Features/LastMinuteBooking.feature

```diff
@@ -0,0 +1,11 @@
+Feature: LastMinuteBooking
+	As a Participant in the hearing 
+  I would like to see a a toast alert when a new participant is added to the hearing
+  So that all the participants are aware that a new participant has been added
+
+@VIH-8055 @HearingTest @Smoketest-Extended
```

Comment:
Can you change the jira tag to a comment, there is little expectation that we will ever run tests based on a jira tag.

Reply:
Done